rfcs/text/0042-risk-score-extensions.md

```diff
@@ -24,16 +21,50 @@ This RFC seeks to extend the [existing risk fields](https://www.elastic.co/guide
     * These categories' definitions may be _extended_ in the future to include new data sources
   * Due to the above category traits, we need to come up with a naming convention for these categorical score fields that allows them to be extended without invalidating the existing field names.
 
-<!--
-Stage 1: If the changes include field additions or modifications, please create a folder titled as the RFC number under rfcs/text/. This will be where proposed schema changes as standalone YAML files or extended example mappings and larger source documents will go as the RFC is iterated upon.
--->
 
 <!--
 Stage X: Provide a brief explanation of why the proposal is being marked as abandoned. This is useful context for anyone revisiting this proposal or considering similar changes later on.
 -->
 
 ## Fields
 
+I'll enumerate the fields being introduced here grouped by their motivation/goal:
+
+### Identifier Fields
+These fields are intended to allow future extensibility of our concept of an "identifier." Currently, we leverage two such fields: `host.name` and `user.name`, to group documents for the purposes of risk scoring, but we definitely imagine this being extended in the future to allow multiple such fields for hosts and users, or more simply to allow administrators to configure fields other than those above. The presence of these two fields allows us to audit/explain how a particular risk score was identified.
+
+* `identifier_field`
+* `identifier_value`
+
+
+### Risk Category Fields
+Some of the context here was discussed in Stage 0; please read the above for that. More specifically, these fields seek to provide the category contributions to the score, and the number of risk inputs in that category, across each of the five proposed categories:
+
+* `category_1_score`
+* `category_1_count`
+* `category_2_score`
+* `category_2_count`
+* `category_3_score`
+* `category_3_score`
+* `category_4_count`
+* `category_4_count`
+* `category_5_score`
+* `category_5_count`
+
+### Risk Explainability
+Beyond the per-category explanations, these fields' purpose is to provide more insight/data for the analyst to further investigate the components of the risk score.
+
+* `riskiest_inputs`
```

Comment:
I think I had originally called this `top_risk_inputs` but decided this was more specific (e.g. what does "top" mean?). However, if we want to change that definition in the future, maybe `top_risk_inputs` or something similar would allow us that flexibility.

Reply:
> maybe `top_risk_inputs` or something similar would allow us that flexibility.

Do you mean 'risk_inputs' will allow us the flexibility? @rylnd

Reply:
No, but `risk_inputs` is certainly more flexible than what I was suggesting!

Reply:
@SourinPaul is that your preference? I have no objections and can make that change.

Reply:
I wish to understand your original comment, which I do now. `riskiest_inputs` serves the purpose, although it still begs the question of relative risk. 
 
 The application using this field (in our case, Entity Analytics) can capture the specifics and not the ECS definition.

Reply:
@SourinPaul I'm personally leaning toward your suggestion of the generic `risk_inputs` for this field name, but I was unable to determine your stance from the previous comment. Could you please clarify?

Reply:
@rylnd I recommend using `risk_inputs` as the ECS field name.